### PR TITLE
Update conditional-access-and-compliance-for-devices.md

### PR DESCRIPTION
**Pull request by jgwmuller**
Added the Microsoft Whiteboard Services as an cloudapp that Team Room devices now can use (GA)

### CODE REVIEW

File: Teams/rooms/conditional-access-and-compliance-for-devices.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ In the example below, the Conditional Access policy works as follows:
     this example, the "Shared devices" group.
 
 2.  The account signing in must only be trying to access Exchange
-    Online, Microsoft Teams, or SharePoint Online. Attempts to sign into
+    Online, Microsoft Teams, SharePoint Online or Microsoft Whiteboard Services. Attempts to sign into
```
Comment (JohanFreelancer9) [inline Teams/rooms/conditional-access-and-compliance-for-devices.md:66]:
```suggestion
    Online, Microsoft Teams, SharePoint Online, or Microsoft Whiteboard Services. Attempts to sign into
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @jgwmuller, kindly commit the minor suggested change. Many thanks.

**Comment by jgwmuller** [comment]
@JohanFreelancer9 commited :-)

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tonysmit 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@tonysmit
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**